### DISCUSSION TITLE
Passkey Information / Metadata

### DISCUSSION THREAD

**Discussion by renchris** [MasterKale/SimpleWebAuthn#546] [General]
Hello. Is there a way to access log-in data of the passkey? For example, a lot of auth providers give information on device type (ex. Macbook), location (city and IP), and browser (ex. Chrome and version).

**Comment by MasterKale** [comment]
Hello @renchris, this project doesn't currently offer any functionality that would help with things like that. How a project using SimpleWebAuthn wishes to identify passkeys to its users is not necessarily a concern of the WebAuthn spec itself, which SimpleWebAuthn is focused on.

That said, I'm debating adding in something to offer a modicum of the ability to nickname passkeys based on common signals seen out of a registration ceremony. It'd involve things like browser user agent parsing, however, and I'm not sure if I want to pull in related dependencies, or spin off a new package in the **@simplewebauthn** namespace to contain that logic. I'm still mulling the idea over, though, so it's still possible I'll go that route.
